packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -53,8 +53,8 @@ class _InsightsHandler {
     }
     private _userCaps?: Capabilities.ResolvedTestrunnerCapabilities = {}
     private listener = Listener.getInstance()
-    private _currentTestId: string | undefined
-    private _cbtQueue: Array<CBTData> = []
+    public _currentTestId: string | undefined
+    public _cbtQueue: Array<CBTData> = []
```

Comment:
If these properties are public they shouldn't start with `_`
```suggestion
    public currentTestId: string | undefined
    public cbtQueue: Array<CBTData> = []
```

Reply:
Removed the _ from these variable names.